checkov/terraform/checks/resource/ncp/LBNetworkPrivate.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class LBNetworkPrivate(BaseResourceCheck):
```

Comment:
@pj991207 @gruebel do we have a base resource value check for NCP? I think it's worth adding it in this PR if not

Reply:
@pj991207 it would be great if you can change the check to be like this one https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/aws/APIGatewayDomainNameTLS.py 
so `get_inspected_key` will be `"network_type"`
and `get_expected_values` should be `"PRIVATE"`
but please test it 😄

Reply:
I fix it. now, the rule works on base resource value check. Thank you for your advice